module/account/src/Application/Form/UserCreateForm.php

```diff
@@ -56,6 +57,10 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ->add(
                 'roleId',
                 RoleIdType::class
+            )
+            ->add(
+                'isActive',
+                CheckboxType::class
```

Comment:
The checbox on front do not send any vale when is unchecked, you need to add default value and false conversion:
```
             ->add(
                'isActive',
                CheckboxType::class,
                [
                    'false_values' => [
                        '0',
                        'false',
                        '',
                        false,
                    ],
                    'empty_data' => false,
                ]
            );
```